code/controllers/mc/master.dm

```diff
@@ -124,15 +124,21 @@ var/CURRENT_TICKLIMIT = TICK_LIMIT_RUNNING
 // 	Make a subsystem, give it the SS_NO_FIRE flag, and do your work in it's Initialize()
 /datum/controller/master/proc/Setup()
 	set waitfor = FALSE
-	sleep(1 SECONDS)
-	to_chat(world, "<span class='boldannounce'>Initializing subsystems...</span>")
-
+	sleep(-1)
+	//moving this random bullshit into here, because it didn't belong in world/New()
```

Comment:
How did you determine these things didn't belong in `world/New()`?

Reply:
world/New contains stuff related to metadata and database stuff, whereas the other stuff that I moved is for in-game stuff, which should be in a subsystem with high priority. However, that would be a big undertaking. As for the other stuff that I moved from master, into more_init, they are low priority